lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -3241,7 +3241,7 @@ public function registerManaged($entity, array $id, array $data)
      * INTERNAL:
      * Clears the property changeset of the entity with the given OID.
      *
-     * @param string $oid The entity's OID.
+     * @param int $oid The entity's OID.
```

Comment:
This was actually a BC break regarding `gedmo/doctrine-extensions` (they adapted the code in https://github.com/doctrine-extensions/DoctrineExtensions/pull/2272 though, so no need to revert it as a second BC break).
this method has `INTERNAL:` in its description, but it is not tagged as `@internal` so using it in third-party code is not flagged. And it looks like there is no public API allowing to achieve that goal.

Reply:
OK, I see that https://github.com/doctrine/orm/pull/9143 actually documents it